sdk/communication/azure-communication-chat/azure/communication/chat/_shared/communication_identifier_serializer.py

```diff
@@ -4,15 +4,17 @@
 # license information.
 # --------------------------------------------------------------------------
 
-from .models import (
+from .._generated.models import (
```

Comment:
This file should be moved out of the shared folder. The shared folder across all packages should be identical. Considering that we are removing the models from the shared folder, we should move this class to the non-shared part of the chat code as well

Reply:
Thanks. Updated in new iteration.